### PR TITLE
Add Trove classifiers for Python 3.10 and 3.11

### PR DESCRIPTION
**Pull request by hugovk**
* Add Trove classifiers for Python 3.10 and 3.11
* Add `python_requires` to help pip
* Fix some typos: Unicode, PyPI and GitHub

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
 setup(
     name='emoji',
     author=author,
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
     author_email=email,
     classifiers=[
         'Development Status :: 4 - Beta',
```
Comment (cvzi) [inline setup.py:37]: Looking at this, I think we should also replace the development status classifier with `Development Status :: 5 - Production/Stable`

Comment (cvzi) [inline setup.py:37]: Looking at this, I think we should also replace the development status classifier with `Development Status :: 5 - Production/Stable`

Comment (hugovk) [reply setup.py]: Updated! :rocket:

**Comment by TahirJalilov** [comment]
Thank you for PR

**Pull request merged**